index.html

```diff
@@ -0,0 +1,72 @@
+<!--
+This page is for development purposes. It is the entry point for `webpack-dev-server`.
+It mocks the html environment for `panel/Index.tsx`. Local Storage simulates `globalState`.
+-->
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="index.css">
+		<style>
+			code { font-family: SF Mono, Menlo, Monaco, 'Courier New', monospace; }
+		</style>
+	</head>
+	<body class="vscode-dark">
+		<div id="root" style="margin: 50px 300px; position: relative;"></div><!-- relative for Popover -->
+		<script>
```

Comment:
Can the scripts here be done in TypeScript or in another javascript file? It would be nice to have a nice separation between HTML and Script.

Reply:
For Typescript, this would require a separate webpack export (aka "pipeline") so I was hesitant.

We can have the Javascript in a separate file, that's no problem.

Reply:
For Typescript, this would require a separate webpack export (aka "pipeline") so I was hesitant.

We can have the Javascript in a separate file, that's no problem.

---
In reply to: [434716503](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r434716503) [](ancestors = 434716503)

Reply:
Moved the bulk of it into "index.js". Note, I left the first script block in as having a separate script file for 9 lines of code seemed overkill. Also note, I couldn't merge it with the other script due to execution order issues.

---
In reply to: [436290240](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r436290240) [](ancestors = 436290240,434716503)